product_docs/docs/epas/15/reference/sql_reference/02_data_types/array_types.mdx

```diff
@@ -34,9 +37,9 @@ CREATE TABLE tictactoe (
 
 However, the current implementation ignores any supplied array size limits, that is, the behavior is the same as for arrays of unspecified length.
 
-The current implementation doesn't enforce the declared number of dimensions either. Arrays of a particular element type are all considered to be of the same type, regardless of size or number of dimensions. So, declaring the array size or number of dimensions in `CREATE TABLE` is simply documentation; it does not affect run-time behavior.
+The current implementation doesn't enforce the declared number of dimensions either. Arrays of a particular element type are all considered to be of the same type, regardless of size or number of dimensions. So, declaring the array size or number of dimensions in `CREATE TABLE` is documentation that doesn't affect runtime behavior.
```

Comment:
I see how you've removed a lot of adverbs like "simply" and those are good improvements, but in this paragraph I think that "simply" makes the meaning more clear. I stumbled over your edit.

Reply:
```suggestion
The current implementation doesn't enforce the declared number of dimensions either. Arrays of a particular element type are all considered to be of the same type, regardless of size or number of dimensions. So, declaring the array size or number of dimensions in `CREATE TABLE` is just documentation and doesn't affect runtime behavior.
```